R/operations.R

```diff
@@ -123,14 +127,23 @@ get_operation_definitions <- function(api, path = NULL) {
 
       # parameters can be defined on path level and overridden on operation
       # level
-      # 
+      #
       # Note that parameters is often a list() rather than NULL, so deal
       # that situation as well.
       if(is.null(operation$parameters) || length(operation$parameters)==0) {
         operation$parameters <- api$paths[[path_name]]$parameters
-      }
-      else {
-        operation$parameters <- c(operation$parameters, api$paths[[path_name]]$parameters)
+      } else {
+        ## check names in operations parameters
```

Comment:
Is the code more readable (maybe not!) with
```
api_parameters <- api$paths[[path_name]]$parameters
```
defined in this code block?

Reply:
I agree, it would look more readable but it is only used in two instances so maybe not worth the extra line..